### DISCUSSION TITLE
Translating RustDesk into Slovene

### DISCUSSION THREAD

**Discussion by BlazOsredkar** [rustdesk/rustdesk#4171] [General]
Hello, I have been reading through your GitHub project and noticed that you are looking for users to help translate RustDesk into other languages. I would be happy to translate it into Slovenian. If it is okay with you, could you please give me some instructions on how to proceed?

Thank you and best regards.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/blob/master/src/lang/sl.rs
